Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -700,3 +702,48 @@ def get_current_exchange():
         return next(iter(exchanges))
     else:
         return DEFAULT_EXCHANGE
+
+
+def change_reference_market_on_config_currencies(old_base_currency: str, new_base_currency: str) -> bool:
+    """
+    Change the base currency from old to new for all configured pair
+    :param old_base_currency:
+    :param new_base_currency:
+    :return: bool, str
+    """
+    success = True
+    message = "Reference market changed on currencies list"
+    try:
+        config_currencies = format_config_symbols(interfaces_util.get_edited_config())
+        regex = rf"/{old_base_currency}$"
+        for currencies_config in config_currencies.values():
+            currencies_config[commons_constants.CONFIG_CRYPTO_PAIRS] = \
+                list(set([re.sub(regex, f"/{new_base_currency}", pair)
```

Comment:
Why a `set` is necessary here?

Reply:
Example : 
start currencies list : BTC/USDT BTC/BUSD
Change reference market from USDT to BUSD
end currencies list : BTC/BUSD x2